pandas/core/generic.py

```diff
@@ -1940,6 +1940,79 @@ def tail(self, n=5):
             return self
         return self.iloc[-n:]
 
+  
+    def sample(self, n=None, frac=None , replace=False, weights=None, seed=None):
+        """
+        Returns a sample of rows from object. 
+        
+        Parameters
+        ----------
+        n: in, optional, Default = 5 if frac = None. 
+            Number of rows to return. Cannot be used with frac.
+        frac: float. Default = None, S
+            Share of rows to return. Cannot be used with n. 
+        replace: boolean. Default = False. 
+            Sample with or without replacement. 
+        weights: Series, list, or ndarray. Default = None.
+            Default 'None' results in equal probability weighting. 
+            If called on a DataFrame or Panel, will also accept the name of a 
+            column as a string. Must be same length as index. 
+            Weights must be normalized. 
+        seed: int. Default = None. 
+            seed to be fed to numpy random.RandomState() Function.
```

Comment:
I would just say `Seed for the random number generator.` The fact that we use numpy.RandomState under the covers is an implementation detail. Eventually, for example, we'll want this to work with groupby -- at which point we'll want to derive a separate seed for every group based on the seed the user provides.